### PR TITLE
feat: add a leaderboard of top contributors

### LINKED ISSUE #5: Add pod stats
Add a way to display the number of contributions between fellows. Make a little leaderboard or similar to motivate people to add new issues!

### PR DESCRIPTION
**Pull request by KhudaDad414**
Resolves #5 
> NOTE: to be merged after #28 

## Please, go through these steps before you submit a PR.
- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

### CODE REVIEW

**Comment by yyjlincoln** [comment]
Resolved a merge conflict - great work!

**Comment by Aniket762** [comment]
@KhudaDad414 just a bit confused I was not able to find the leaderboard in the web UI. Just a suggestion using Github API instead of Github Actions could have been an easier approach and you could have fetched more params in total.

File: .github/workflows/contribution-count.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Contribution Count
+
+on:
+  push:
+  issues:
```
Comment (Aniket762) [inline .github/workflows/contribution-count.yml:5]: This is a good approach but I think you should consider other params as well to build the leaderboard :)

Comment (KhudaDad414) [reply .github/workflows/contribution-count.yml:5]: Hey, just wanted to let you know that this PR is a draft, Draft is a PR that is not ready for review. thanks for your suggestion though.

Comment (Aniket762) [reply .github/workflows/contribution-count.yml:5]: Oops! sorry I missed that it was drafted!

**Comment by KhudaDad414** [comment]
@Aniket762 why we are not using GitHub API? well, there are two reasons.

## Security
Since we have no backend and the website is statically generated, we have to put the Github Token on the page itself, which anybody can inspect the page and pick it up which is a risk.

## GitHub API call limit
Github only allocates 5000 points per hour which if we want to scrap the whole repo each time someone opens the page would consume those points like butter.

**Pull request merged**